src/main/java/org/carlmontrobotics/lib199/swerve/SwerveModule.java

```diff
@@ -83,7 +94,8 @@ public SwerveModule(SwerveConfig config, ModuleType type, CANSparkMax drive, CAN
         final double neoStallCurrentAmps = 166;
         double currentLimitAmps = neoFreeCurrentAmps + 2*motorTorqueLimitNewtonMeters / neoStallTorqueNewtonMeters * (neoStallCurrentAmps-neoFreeCurrentAmps);
         // SmartDashboard.putNumber(type.toString() + " current limit (amps)", currentLimitAmps);
-        drive.setSmartCurrentLimit((int)Math.min(50, currentLimitAmps));
+        drive.configure(new SparkMaxConfig().smartCurrentLimit(Math.min(50,(int)currentLimitAmps)),
```

Comment:
This can just be added to the running config
```suggestion
        driveConfig.smartCurrentLimit(Math.min(50,(int)currentLimitAmps));
```